### PR TITLE
[Feature] Show dates on exam cover, when the exam spans multiple days

### LINKED ISSUE #3691: Different Date support for Exam Time
### Describe the bug

The welcome page of the exam does not support that the date for the submission is on a different date than for the start.
Only the start date and times are displayed, but the fact that the second time is on a different date is not immediately obvious.
In addition, the displayed duration for the exam should be scaled up accordingly.
For example on hours and days.

### To Reproduce

1. Create an exam with > 24 hours editing time for students
2. Look at the intro page

### Expected behavior

Time shall show start and end date iff start and end date are different

### Screenshots

image

### What browsers are you seeing the problem on?

Firefox

### Additional context

_No response_

### Relevant log output

_No response_

### PR DESCRIPTION
**Pull request by JohannesStoehr**
### Checklist
- [x] I tested *all* changes and *all* related features with different users (student, tutor, editor, instructor, admin) on the test server https://artemistest.ase.in.tum.de.
- [x] Client: I followed the [coding and design guidelines](https://docs.artemis.ase.in.tum.de/dev/guidelines/client/).
- [x] Client: I added multiple screenshots/screencasts of my UI changes
- [x] Client: I translated all newly inserted strings into English and German.

### Motivation and Context

Fix #3691 

### Description

Shows the date of start and end time of an exam on the cover, when they are different.

### Steps for Testing

1. Log in to Artemis
2. Create two exams. One that starts and ends on the same day, another that spans multiple days
3. Go to the start page of these exams as a student.
4. Check that the dates and times are displayed correctly and are intuitively

### Screenshots

An exam that only lasts for one day:
Bildschirmfoto 2021-07-07 um 22 38 10

An exam that lasts multiple days:
Bildschirmfoto 2021-07-07 um 22 35 33

### CODE REVIEW

**Review by MDK5 — APPROVED** [review]
Tested on ts1, looked good.

Code changes also look good to me

**Review by Malyuk-A — APPROVED** [review]
Tested on TS3. Works as described & code looks fine👍

Just one small idea : 
What do you think about changing 
Duration(mm:ss): 1441:00 
to 
Duration(hh:mm:ss): 24:01:00
Because I think that it should be intuitive to understand how long the duration is without calculating anything in your head. (Especially for long exams)

File: src/main/webapp/app/exam/participate/information/exam-information.component.ts [hunk 1]
```diff
@@ -23,4 +23,18 @@ export class ExamInformationComponent {
...
+            return false;
+        }
+        let endDate;
+        if (this.studentExam && this.studentExam.workingTime && this.exam.startDate) {
+            endDate = moment(this.exam.startDate).add(this.studentExam.workingTime, 'seconds');
+        } else {
+            endDate = this.exam.endDate;
+        }
+
+        return endDate.dayOfYear() !== this.exam.startDate.dayOfYear();
```
Comment (dfuchss) [inline src/main/webapp/app/exam/participate/information/exam-information.component.ts:38]: I think there might be the unrealistic (!) edge case of an exam that lasts exactly a year (which is of course very unrealistic :D)

**Review by dfuchss — APPROVED** [review]
Code looks good to me!

**Review by lschlesinger — APPROVED** [review]
Code and tests look good!

**Review by willergeorg — APPROVED** [review]
Tested on TS1. Looks good!

**Comment by dfuchss** [comment]
> Just one small idea :
> What do you think about changing
> Duration(mm:ss): 1441:00
> to
> Duration(hh:mm:ss): 24:01:00
> Because I think that it should be intuitive to understand how long the duration is without calculating anything in your head. (Especially for long exams)

I think your idea is very good! I've created a separate PR for such a suggestion 👍 
#3692

**Review by ArtemTsurikov — APPROVED** [review]
Looks good on TS1

**Review by NicolasUllmann — APPROVED** [review]
Tested on TS1. Everything worked as expected.

**Review by real-dmy — APPROVED** [review]
Tested on TS1. Looks good.

**Pull request merged**